### PR TITLE
Spelling and phrasing tweaks

### PR DESCRIPTION
**Pull request by mahmoud**
Very cool project. I made a few spelling and phrasing improvements which I hope and strongly suspect don't change any legal meaning.

I did have a few questions about the actual content, but I'll save those for issues :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 
-an tiny contributor license agreement for public-private licensing projects
+A tiny contributor license agreement for public-private licensing projects.
 
 Download the most recent release from [GitHub](https://github.com/indieopensource/tiny-cla/releases), or read the [current development version of the terms in Markdown format](https://github.com/indieopensource/tiny-cla/blob/master/cla.md).
 
 ## Be Warned!
 
-**Licenses are prescription-strength legal devices.  If you need terms, don't be an dufus.  Hire a lawyer.  A good one will ask good questions. They may decide this form fits your needs.**
+**Licenses are prescription-strength legal devices.  If you need terms, don't be an doofus.  Hire a lawyer.  A good one will ask good questions. They may decide this form fits your needs.**
```
Comment (kemitchell) [inline README.md:9]: Hah! Apparently M-W lists "dufus" as a "less common spelling"!

File: README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 # indieopensource.com Tiny Contributor License Agreement
 
-an tiny contributor license agreement for public-private licensing projects
+A tiny contributor license agreement for public-private licensing projects.
```
Comment (kemitchell) [inline README.md:3]: Hmm. Not a complete sentence.

File: cla.md [hunk 3]
```diff
@@ -2,6 +2,6 @@
 
 Version [version]
 
-I, {{{contributor name}}}, give {{{project steward name}}} permission to license my contributions on any terms they like.  I am giving you this license in order to make it possible for you to accept my contributions into the project.
+I, {{{contributor name}}}, give {{{project steward name}}} permission to license my contributions on any terms they like.  I am giving them this license in order to make it possible for them to accept my contributions into the project.
 
-***As far as the law allows, my contributions comes as is, without any warranty or condition, and I will be liable to anyone for any damages related to this software or this license, under any kind of legal claim.***
+***As far as the law allows, my contributions come as is, without any warranty or condition, and I will be liable to anyone for any damages related to this software or this license, under any kind of legal claim.***
```
Comment (kemitchell) [inline cla.md:7]: :+1:

File: cla.md [hunk 4]
```diff
@@ -2,6 +2,6 @@
 
 Version [version]
 
-I, {{{contributor name}}}, give {{{project steward name}}} permission to license my contributions on any terms they like.  I am giving you this license in order to make it possible for you to accept my contributions into the project.
+I, {{{contributor name}}}, give {{{project steward name}}} permission to license my contributions on any terms they like.  I am giving them this license in order to make it possible for them to accept my contributions into the project.
```
Comment (kemitchell) [inline cla.md:5]: Yes, this is better. The steward could be an individual or a company.

File: README.md (hunk 2 above)
Comment (kemitchell) [reply README.md:3]: Might I ask you to revert this change?

**Review by kemitchell — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mahmoud) [reply README.md:3]: The capitalization and punctuation were more reflexive side effects of editing the "an tiny" to "a tiny".

I was going to ask about this lead, anyways. I'm not sure what public-private licensing means and I think maybe a link would be in order.

Also should we just make the lead/lede complete? "Tiny CLA is a small but sufficient contributor license agreement for public-private licensing projects." Or something like that?

Comment (kemitchell) [reply README.md:3]: Complete sentence? Go for it!

**Comment by kemitchell** [comment]
@mahmoud, thanks again for this!

Following up on your question about "public-private licensing": That's a new term that I want to establish, through indieopensource.com, as an alternative to "dual licensing". People have said "dual licensing" for a long time, but meant several different things by it, which causes confusion.

I went ahead and added a link to indieopensource.com in `README`: 534a92887f2be95db19eda971264edc24d4a82a4

**Pull request merged**